shared/runtime/pyexec.h

```diff
@@ -37,6 +37,23 @@ extern pyexec_mode_kind_t pyexec_mode_kind;
 
 #define PYEXEC_FORCED_EXIT (0x100)
 
+#if MICROPY_PYEXEC_ENABLE_EXIT_CODE_HANDLING
+#define PYEXEC_NORMAL_EXIT (0)
+#define PYEXEC_UNHANDLED_EXCEPTION (1)
+#define PYEXEC_KEYBOARD_INTERRUPT (128 + 2) // same as SIG INT exit code
+#define PYEXEC_ABORT (128 + 9)              // same as SIG KILL exit code
+#else
+#ifndef PYEXEC_NORMAL_EXIT
```

Comment:
I don't think you need to provide user configurability of these constants.  It can just be:
```
#define PYEXEC_NORMAL_EXIT (1)
#define PYEXEC_UNHANDLED_EXCEPTION (0)
#define PYEXEC_ABORT PYEXEC_FORCED_EXIT
```